api/tests/unit/domain/usecases/get-campaign-participation_test.js

```diff
@@ -1,28 +1,72 @@
-const { expect, sinon, domainBuilder } = require('../../../test-helper');
+const { expect, sinon, domainBuilder, catchErr } = require('../../../test-helper');
 const getCampaignParticipation = require('../../../../lib/domain/usecases/get-campaign-participation');
+const { UserNotAuthorizedToAccessEntity } = require('../../../../lib/domain/errors');
 
 describe('Unit | UseCase | get-campaign-participation', () => {
 
   const options = {};
 
-  let campaignParticipationRepository;
+  let campaignParticipationRepository, campaignRepository;
 
   beforeEach(() => {
     campaignParticipationRepository = {
       get: sinon.stub(),
     };
+    campaignRepository = {
+      checkIfUserOrganizationHasAccessToCampaign: sinon.stub()
+    };
+  });
+
+  context('when user is the user of the campaignParticipation', () => {
+    it('should get the campaignParticipation', async () => {
+      // given
+      const campaignParticipationId = 1;
+      const userId = 1;
+      const campaignParticipation = domainBuilder.buildCampaignParticipation({ id: campaignParticipationId, userId });
+      campaignParticipationRepository.get.withArgs(campaignParticipationId, options).resolves(campaignParticipation);
+      campaignRepository.checkIfUserOrganizationHasAccessToCampaign.withArgs(campaignParticipation.campaignId, userId).resolves(false);
+
+      // when
+      const result = await getCampaignParticipation({ campaignParticipationId, options, campaignParticipationRepository, campaignRepository, userId });
+
+      // then
+      expect(result).to.equal(campaignParticipation);
+    });
   });
 
-  it('should get the campaignParticipation', async () => {
-    // given
-    const campaignParticipationId = 1;
-    const campaignParticipation = domainBuilder.buildCampaignParticipation({ id: campaignParticipationId });
-    campaignParticipationRepository.get.withArgs(campaignParticipationId, options).resolves(campaignParticipation);
+  context('when user is the organization of the campaignParticipation', () => {
```

Comment:
"when user is a member of the organization which is related to requested campaignParticipation" ?